tox.ini

```diff
@@ -3,6 +3,9 @@ envlist = py27, py34, py35, py36, py37, pypy, pypy3, py27-flake8, py37-flake8, i
 skip_missing_interpreters = True
 
 [testenv]
+setenv =
+    PYTHONPATH = {toxinidir}
+deps = -r{toxinidir}/test-requirements.txt
 commands =
     pip install -r test-requirements.txt
```

Comment:
I don't think we still need this command if we're specifying the dependencies above.

Reply:
Do you mean `commands = pip install -r test-requirements.txt`?

Reply:
Yes. I believe using `deps` implies a `pip install` command.